<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BirthdayListEventsConfig</class>
 <widget class="QWidget" name="BirthdayListEventsConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>549</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="21" column="4">
    <spacer name="spacerScreenRight">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="17" column="1">
    <widget class="QCheckBox" name="chckNamedayDateField">
     <property name="text">
      <string>Date field:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="2">
    <widget class="QLabel" name="lblComingShowDays_2">
     <property name="text">
      <string comment="label: show events within x &lt;days&gt;">days</string>
     </property>
    </widget>
   </item>
   <item row="21" column="3">
    <widget class="QLineEdit" name="lineEditNamedayCustomDateField">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="29" column="1">
    <widget class="QCheckBox" name="chckShowAnniversaries">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="showAnniversaries" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="27" column="1">
    <spacer name="spacerNamedaysBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="lblNamedayDisplayMode">
     <property name="text">
      <string>Nameday display mode:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="26" column="1" colspan="3">
    <widget class="QRadioButton" name="rbNamedayShowAllFromCal">
     <property name="text">
      <string>All names from calendar</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bgNamedayDisplayMode</string>
     </attribute>
    </widget>
   </item>
   <item row="24" column="1" colspan="3">
    <widget class="QRadioButton" name="rbNamedayShowAggrEvents">
     <property name="text">
      <string>Aggregated events</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bgNamedayDisplayMode</string>
     </attribute>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblTitleNamedays">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Namedays</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="spinComingShowDays">
     <property name="maximum">
      <number>335</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblPastShowDays">
     <property name="text">
      <string>Show events within last:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSpinBox" name="spinPastShowDays">
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <spacer name="spacerNamedaysBottom_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="25" column="1" colspan="3">
    <widget class="QRadioButton" name="rbNamedayShowIndEvents">
     <property name="text">
      <string>Individual events</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bgNamedayDisplayMode</string>
     </attribute>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="lblNamedayCalendar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Nameday calendar:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QComboBox" name="cmbNamedayCalendar"/>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="lblNamedayIdentifyBy">
     <property name="text">
      <string>Identify by:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="21" column="2">
    <widget class="QRadioButton" name="rbNamedayDateFieldCustom">
     <property name="text">
      <string>Custom:</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bgNamedayDateFieldTypes</string>
     </attribute>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblComingShowDays">
     <property name="text">
      <string>Show events within next:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="28" column="0">
    <widget class="QLabel" name="lblTitleAnniversaries">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Anniversaries</string>
     </property>
    </widget>
   </item>
   <item row="35" column="1">
    <spacer name="spacerScreenBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lblShowNamedays">
     <property name="text">
      <string>Show namedays:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="QCheckBox" name="chckShowNamedays">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="29" column="0">
    <widget class="QLabel" name="lblShowAnniversaries">
     <property name="text">
      <string>Show anniversaries:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QCheckBox" name="chckNamedayNameField">
     <property name="text">
      <string>Name field:</string>
     </property>
    </widget>
   </item>
   <item row="32" column="1">
    <widget class="QRadioButton" name="rbAnniversaryDateFieldAnniversary">
     <property name="text">
      <string>Anniversary</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bgAnniversaryDateFieldTypes</string>
     </attribute>
    </widget>
   </item>
   <item row="32" column="0">
    <widget class="QLabel" name="lblAnniversaryField">
     <property name="text">
      <string>Date field:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="34" column="1">
    <widget class="QRadioButton" name="rbAnniversaryDateFieldCustom">
     <property name="text">
      <string>Custom:</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bgAnniversaryDateFieldTypes</string>
     </attribute>
    </widget>
   </item>
   <item row="34" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditAnniversaryCustomDateField"/>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QLabel" name="lblPastShowDays_2">
     <property name="text">
      <string comment="label: show events within x &lt;days&gt;">days</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2" colspan="2">
    <widget class="QRadioButton" name="rbNamedayDateFieldAnniversary">
     <property name="text">
      <string>Anniversary</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">bgNamedayDateFieldTypes</string>
     </attribute>
    </widget>
   </item>
   <item row="23" column="2" colspan="2">
    <widget class="QComboBox" name="cmbNamedayNameField">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Given name</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinComingShowDays</tabstop>
  <tabstop>spinPastShowDays</tabstop>
  <tabstop>chckShowNamedays</tabstop>
  <tabstop>cmbNamedayCalendar</tabstop>
  <tabstop>chckNamedayDateField</tabstop>
  <tabstop>rbNamedayDateFieldAnniversary</tabstop>
  <tabstop>rbNamedayDateFieldCustom</tabstop>
  <tabstop>lineEditNamedayCustomDateField</tabstop>
  <tabstop>chckNamedayNameField</tabstop>
  <tabstop>cmbNamedayNameField</tabstop>
  <tabstop>rbNamedayShowAggrEvents</tabstop>
  <tabstop>rbNamedayShowIndEvents</tabstop>
  <tabstop>rbNamedayShowAllFromCal</tabstop>
  <tabstop>chckShowAnniversaries</tabstop>
  <tabstop>rbAnniversaryDateFieldAnniversary</tabstop>
  <tabstop>rbAnniversaryDateFieldCustom</tabstop>
  <tabstop>lineEditAnniversaryCustomDateField</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chckShowAnniversaries</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditAnniversaryCustomDateField</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowAnniversaries</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblAnniversaryField</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblNamedayIdentifyBy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbNamedayNameField</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblNamedayCalendar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbNamedayCalendar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>lblNamedayDisplayMode</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbNamedayShowAggrEvents</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbNamedayShowIndEvents</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbNamedayShowAllFromCal</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckNamedayDateField</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbNamedayDateFieldAnniversary</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckNamedayDateField</sender>
   <signal>toggled(bool)</signal>
   <receiver>rbNamedayDateFieldCustom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckNamedayNameField</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbNamedayNameField</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>chckNamedayDateField</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chckShowNamedays</sender>
   <signal>toggled(bool)</signal>
   <receiver>chckNamedayNameField</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbAnniversaryDateFieldCustom</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditAnniversaryCustomDateField</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bgNamedayDateFieldTypes"/>
  <buttongroup name="bgNamedayDisplayMode"/>
  <buttongroup name="bgAnniversaryDateFieldTypes"/>
 </buttongroups>
</ui>
